packages/site/src/components/PaymentChannelDetails.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { ChannelStatus } from "@statechannels/nitro-rpc-client/src/types";
+
+import PaymentChannelDetails from "./PaymentChannelDetails";
+
+const meta: Meta<typeof PaymentChannelDetails> = {
+  title: "PaymentChannelDetails",
+  component: PaymentChannelDetails,
+};
+
+export default meta;
+
+type Story = StoryObj<typeof PaymentChannelDetails>;
+
+export const PaymentChannelDetailsComponent: Story = {
+  render: () => (
```

Comment:
Nit, to me, it's cleaner to use args instead of `render`. Check out https://github.com/statechannels/nitro-gui/blob/9dbdffbee88e4e38f5aca0228e19ac5a86b1ddc2/packages/site/src/components/RpcConnect.stories.tsx#L15.